File: app/src/main/java/com/ufpb/getha/ui/gallery/CalculadoraFragment.java
package com.ufpb.getha.ui.gallery;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.lifecycle.ViewModelProvider;

import com.ufpb.getha.databinding.FragmentCalculadoraBinding;

public class CalculadoraFragment extends Fragment {

    private FragmentCalculadoraBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        CalculadoraViewModel calculadoraViewModel = new ViewModelProvider(this).get(CalculadoraViewModel.class);

        binding = FragmentCalculadoraBinding.inflate(inflater, container, false);

        View root = binding.getRoot();

        final TextView textView = binding.textGallery;

        calculadoraViewModel.getText().observe(getViewLifecycleOwner(), textView::setText);

        return root;
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();

        binding = null;
    }
}

File: app/src/main/java/com/ufpb/getha/ui/gallery/CalculadoraFragment.kt
package com.ufpb.getha.ui.gallery

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.lifecycle.ViewModelProvider
import com.ufpb.getha.databinding.FragmentCalculadoraBinding

class CalculadoraFragment : Fragment() {
    private var binding: FragmentCalculadoraBinding? = null
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val calculadoraViewModel = ViewModelProvider(this).get(
            CalculadoraViewModel::class.java
        )
        binding = FragmentCalculadoraBinding.inflate(inflater, container, false)
        val root: View = binding!!.getRoot()
        val textView = binding!!.textGallery
        calculadoraViewModel.text.observe(getViewLifecycleOwner()) { text: CharSequence? ->
            textView.text = text
        }
        return root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        binding = null
    }
}